gemsfx/src/main/java/com/dlsc/gemsfx/BeforeAfterView.java

```diff
@@ -14,27 +19,21 @@
 import javafx.scene.image.Image;
 import javafx.scene.image.ImageView;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 import java.util.Objects;
 
 
 public class BeforeAfterView extends Control {
 
-    private static final PseudoClass horizontal = PseudoClass.getPseudoClass("horizontal");
-    private static final PseudoClass vertical = PseudoClass.getPseudoClass("vertical");
+    private static final Orientation DEFAULT_ORIENTATION = Orientation.HORIZONTAL;
+    private static final PseudoClass PSEUDO_CLASS_HORIZONTAL = PseudoClass.getPseudoClass("horizontal");
+    private static final PseudoClass PSEUDO_CLASS_VERTICAL = PseudoClass.getPseudoClass("vertical");
 
     public BeforeAfterView() {
         getStyleClass().add("before-after-view");
 
-        Label before = new Label();
```

Comment:
Why did you remove this? This is not test code, this is the content shown by the view when the application hasn't specified anything. I think this makes sense to have so that it is obvious how the control works. Your thoughts?

Reply:
I've just updated the Default node. Please review